### DISCUSSION TITLE
htmx

### DISCUSSION THREAD

**Discussion by gassechen** [rabbibotton/clog#308] [Ideas]
Hello, what do you think of this bookstore and this concept?
[https://htmx.org/](https://htmx.org/)
Could it be incorporated into CLOG?

**Comment by K1D77A** [comment]
As per https://github.com/ruricolist/spinneret/issues/88
You can use Spinneret to generate HTMX

**Reply by gassechen** [reply]
If I am using spinneret with htmx, can I use it with clog?

**Reply by K1D77A** [reply]
```lisp
(clog:create-div <clog-obj> :content (spinneret:with-html-string (:h1 "Hi"))) 
```

**Reply by aykaramba** [reply]
All Common Lisp libraries are available to you in Clog, just like in that example.  Cool, right?

**Reply by gassechen** [reply]
Great!!
